src/gasman.c

```diff
@@ -1099,6 +1099,27 @@ static void MaybeMoveBags(void)
 
 #endif
 
+/****************************************************************************
+**
+*F  OutOfMemoryPanic(size) . . . . . . . . . .  handle running out of memory
```

Comment:
```suggestion
*F  OutOfMemoryPanic(size, reason) . . . . . . . handle running out of memory
```

Reply:
The comment is also in the wrong spot, i.e., not directly in front of the function